src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -128,5 +128,9 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "DELETE" }
     ),
+    usersLoader: gravityLoader(
+      "users",
+      {}
```

Comment:
Ah, in this case you don't even need this `{}`. It was just a 'placeholder' for if you needed to pass an option in to the last param.